main.js

```diff
@@ -2,3 +2,52 @@
 // Go back to the lesson if you feel lost
 // Read the docs well, they usually have pretty well documented examples
 // Consider making your fetch work with a button
+
+
+
+const URL = 'https://rickandmortyapi.com/api/character/';
+
+function getInfo() {
+    axios({ method: 'get', url: URL })
+        .then(res => {
+            // WHEN SUCCESS
+            // console.log(res.data);
```

Comment:
you can also create new elements here then append them into container.

so your HTML code looks cleaner and less messy.